### PR TITLE
change quote type at git-branch page

### PR DESCRIPTION
**Pull request by conao3**
I found broken display at `git-branch` page.
It seems that the asterisk is surrounded by code blocks, but it is not well recognized and misunderstood as a chapter break.
Converted to single quotes, but more fundamental changes may be required

Screenshot from 2019-10-19 15-56-41

---
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by mebeim** [comment]
> It seems that the asterisk is surrounded by code blocks, but it is not well recognized and misunderstood as a chapter break.

It is not well recognized **by the client**. This is a client bug, not a problem with the page itself. Which client are you using? The client you're using seems to have parsing issues, even the last description line is not being rendered correctly. I would suggest closing this and opening an issue on the client's GitHub page.

**Comment by conao3** [comment]
OK, thanks for your suggestion! I will.

P.S.
How can I find out what I am using as a client? I am using `tldr` on Ubuntu-bionic and installed it with `apt install tldr`.
`tldr --version` outputs 0.2.3.

**Comment by agnivade** [comment]
See the file type of the tldr file. That should give you some indication.

**Comment by conao3** [comment]
This is a result. Well, I am using Haskell `tldr`...??
```
conao-macbook:~ conao
$ which tldr
/usr/bin/tldr


conao-macbook:~ conao
$ type tldr
tldr は /usr/bin/tldr です


conao-macbook:~ conao
$ file $(which tldr)
/usr/bin/tldr: symbolic link to /etc/alternatives/tldr


conao-macbook:~ conao
$ file /etc/alternatives/tldr
/etc/alternatives/tldr: symbolic link to /usr/bin/tldr-hs


conao-macbook:~ conao
$ file /usr/bin/tldr-hs 
/usr/bin/tldr-hs: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/l, for GNU/Linux 3.2.0, BuildID[sha1]=4578eee0673488303e6ac17acda29f78f3a26ee3, stripped
```

**Comment by agnivade** [comment]
Seems like so. Please file a bug on https://github.com/psibi/tldr-hs

**Comment by conao3** [comment]
OK, Thanks!
